src/main/deltachat.js

```diff
@@ -278,26 +279,31 @@ class DeltaChatController {
   /**
    * Dispatched from menu alternative in SplittedChatListAndView
    */
-  leaveGroup (chatId) {
+  leaveGroup (chatId) {selectChat
     log('leaving chat', chatId)
     this._dc.removeContactFromChat(chatId, C.DC_CONTACT_ID_SELF)
   }
 
-  markNoticedChat (chatId) {
-    log('mark noticed chat', chatId)
-    this._dc.markNoticedChat(chatId)
+  selectChat(chatId) {
+    log('selecting chat with id', chatId)
+    this._selectedChatId = chatId
+    electron.ipcMain.emit('render')
+    log(electron)
   }
 
   /**
    * Returns the state in json format
    */
   render () {
+    let chats = this._chats()
+
     return {
       configuring: this.configuring,
       credentials: this.credentials,
       ready: this.ready,
-      chats: this._chats(),
-      contacts: this._contacts()
+      chats: chats,
```

Comment:
Can do just `chats` here.

Reply:
or `chats: this._chats()` which would match the below contacts()